packages/react-core/src/components/CalendarMonth/CalendarMonth.tsx

```diff
@@ -90,7 +90,7 @@ export const isValidDate = (date: Date) => Boolean(date && !isNaN(date as any));
 const today = new Date();
 
 export const CalendarMonth = ({
-  date: dateProp = today,
+  date: dateProp,
```

Comment:
Shouldn't this be today or defaultDate?

Reply:
Nicole and I had a quick convo regarding that. Essentially this more follows how it works within the Date Picker component, where if no actual date is passed in then no date is "selected".

Reply:
It looks like the component was already defaulting the date to "today" even though it is required (I am not sure why, possibly an oversight).  The ask in the original issue was to default to "today" if no date is provided.

@mcarrano do you have a n opinion on this?

Reply:
If we think of the DatePicker like a form element, it seems like always needing a date selected is not ideal. I'm imagining, for example, a toolbar filter which includes a datepicker, or inline calendarMonth. the user may not want a date to always be applied to the filterable data?

Reply:
I don't feel strongly about this one, but I agree there may be instances where having a date selected by default does not make sense.

Reply:
Ok.  I am fine with that, @thatblindgeye,  we should update the original epic issue to say why we are not defaulting to "today" since that was the ask.